src/turndown.js

```diff
@@ -358,6 +353,24 @@ turndownService.addRule("h4", {
   }
 })
 
+turndownService.addRule("youtube_shortcodes", {
+  filter: (node, options) => {
+    const nodeClass = node.getAttribute("class")
+    if (
+      node.nodeName === "DIV" &&
+      nodeClass &&
+      nodeClass.includes(YOUTUBE_SHORTCODE_PLACEHOLDER_CLASS)
+    ) {
+      return true
+    }
+    return false
```

Comment:
wouldn't this be equivalent to 

```js
return (
  node.nodeName === "DIV" && nodeClass === YOUTUBE_SHORTCODE_PLACEHOLDER_CLASS
)
```

just slightly more compact that way I suppose